libraries/networking/src/FileResourceRequest.cpp

```diff
@@ -18,17 +18,15 @@ void FileResourceRequest::doSend() {
     
     QFile file(filename);
     
-    _state = Finished;
-    if (file.exists()) {
-        if (file.open(QFile::ReadOnly)) {
-            _data = file.readAll();
-            _result = ResourceRequest::Success;
-        } else {
-            _result = ResourceRequest::AccessDenied;
-        }
+    if (!file.exists()) {
+        _result = NotFound;
+    } else if (file.open(QFile::ReadOnly)) {
+        _data = file.readAll();
+        _result = Success;
     } else {
-        _result = ResourceRequest::NotFound;
+        _result = AccessDenied;
     }
```

Comment:
I had originally written it the way @Atlante45  made it in this PR and decided to write it the longer way since I found it easier to parse what the conditionals were doing. This is potentially a better setup for branching on the CPU though right?